### DISCUSSION TITLE
Playing with an all verified mods profile tags the lobby as "[MODDED]"

### DISCUSSION THREAD

**Discussion by ReLLiKMaster** [trumank/mint#260] [General]
As far as I've read that shouldn't be the case with newer MINT versions. Did an update change that recently?

**Comment by ReLLiKMaster** [comment]
![image](https://github.com/user-attachments/assets/83f96f2c-3ce9-49ee-9feb-1575e9d5674f)
![image](https://github.com/user-attachments/assets/9e4f4f31-f966-4c13-9672-ff76f8ba7acb)

These are the mods I'm running currently and this is how it looks like in the server browser: 
![image](https://github.com/user-attachments/assets/296aebf0-fd88-4671-8e3a-bdfaccffe831)

**Comment by ReLLiKMaster** [comment]
I've uninstalled the mods, reinstalled them, manually wiped the pak, deleted all sandbox and approved mods and reinstalled MINT but to no avail sadly

**Comment by Wepeell** [comment]
This was never implemented in the master branch as far as I can tell. https://github.com/trumank/mint/pull/145
